### PR TITLE
Add doc URL to profile format and use it display help link.

### PR DESCRIPTION
**Pull request by ghemawat**
image

### CODE REVIEW

**Comment by ghemawat** [comment]
This is now ready to review. Changes from prior commit:
1. Rebased to pick up Go version changes.
2. Added a small test to cover new code.

**Review by aalexand — APPROVED** [review]

File: proto/profile.proto [hunk 1]
```diff
@@ -93,6 +93,9 @@ message Profile {
   // Index into the string table of the type of the preferred sample
   // value. If unset, clients should default to the last sample value.
   int64 default_sample_type = 14;
+  // Documentation link for this profile. May be missing if the profile was
+  // generated by older code or code that did not bother to supply a link.
```
[the remark this answers is not in the document]
Comment (ghemawat) [reply proto/profile.proto:97]: Done

**Comment by ghemawat** [comment]
Will need another approval because of the comment change, sorry.

**Review by aalexand — APPROVED** [review]

**Comment by ghemawat** [comment]
> @ghemawat We didn't add printing the link in the pprof CLI output, right? Should we add that too? E.g. the `-top` output has some summary information at the beginning, perhaps we should show the link there?

Good point. I can do that in a bit. It is less useful since it doesn't show up as a clickable link, but might as well add it.

**Pull request merged**